data_services/deploy/ansible/Vagrantfile

```diff
@@ -0,0 +1,22 @@
+# -*- mode: ruby -*-
+# vi: set ft=ruby :
+Vagrant.configure("2") do |config|
+  config.vm.box = "trusty64server"
+  config.vm.box_url = "https://oss-binaries.phusionpassenger.com/vagrant/boxes/latest/ubuntu-14.04-amd64-vbox.box"
+  config.ssh.username = 'vagrant'
+  config.ssh.forward_agent = true
+
+  config.vm.define :production do |production|
+  	production.vm.network :public_network, :bridge => 'eth0', :auto_config => false
```

Comment:
Thanks @mbertrand. Makes sense. Although I am bit surprised because at one point :8000 work but it may because of some other config changes I made at the time. I think private IP sounds good to me. On this note, should we host this at Epidemico (that would be my preference) or Kitware (specially for GUMBO as we develop new data layers).